WordPress/Classes/ViewRelated/Reader/ReaderSiteSearchViewController.swift

```diff
@@ -0,0 +1,386 @@
+import UIKit
+
+/// Displays search results from a reader site search.
+///
+class ReaderSiteSearchViewController: UITableViewController {
+
+    // MARK: - Properties
+    // MARK: Table / Sync Handlers
+
+    fileprivate lazy var tableHandler: WPTableViewHandler = {
+        let tableHandler = WPTableViewHandler(tableView: self.tableView)
+        return tableHandler
+    }()
+
+    fileprivate lazy var syncHelper: WPContentSyncHelper = {
+        let syncHelper = WPContentSyncHelper()
+        syncHelper.delegate = self
+        return syncHelper
+    }()
+
+    // MARK: Data
+
+    fileprivate var feeds: [ReaderFeed] = [] {
+        didSet {
+            reloadData()
+        }
+    }
+    fileprivate var totalFeedCount: Int = 0
+
+    var searchTerm: String? = nil {
+        didSet {
+            feeds = []
+            totalFeedCount = 0
+
+            syncHelper.syncContentWithUserInteraction(false)
+        }
+    }
+
+    // MARK: Views
+
+    fileprivate let statusView: WPNoResultsView = {
+        let view = WPNoResultsView()
```

Comment:
As you identified, I went with `WPNoResultsView` for now so that it matched the existing `ReaderStreamViewController`, as they're displayed in the same parent view.